tests/tests_C2_economic_functions.py

```diff
@@ -0,0 +1,86 @@
+import src.C2_economic_functions as e_functions
+
+project_life = 20
+wacc = 0.1
+investment_t0 = 220000
+tax = 0.15
+lifetime = [20, 15]
+present_value = 295000
+fuel_keys = {
+    "fuel_price": 1.3,
+    "fuel_price_change_annual": 0,
+    "project_lifetime": project_life,
+    "wacc": wacc
+}
+
+
+def test_annuity_factor_calculation():
+    """
+
+    Tests whether the MVS is correctly calculating the annuity factor
+    """
+    AF = e_functions.annuity_factor(project_life, wacc)
+    assert AF == 1 / 0.1 - 1 / (0.1 * (1 + 0.1) ** 20)
+
+
+def test_crf():
+    """
+
+    Tests whether the MVS is correctly calculating the capital recovery factor
+    """
+    CRF = e_functions.crf(project_life, wacc)
+    assert CRF == (0.1 * (1 + 0.1) ** 20) / ((1 + 0.1) ** 20 - 1)
+
+
+def test_capex_from_investment(lifetime):
```

Comment:
I'd suggest to split this test function into two with f. e. the following names:
`test_capex_from_investment_project_life_equals_lifetime()` --> test with `project_life` = `lifetime`
`test_capex_from_investment_lifetime_smaller_than_project_life()`  --> test with `lifetime` < `project_life`

Reply:
The reason for this suggestion is that if only one of the cases fails you already know which one. Apart from that it's convenient to show in the function name what you are testing as the function name will be shown when the test fails but the docstring won't.

Reply:
We also need to add a `test_capex_from_investment_project_life_larger_than_project_life()` - this is eg. the case for infrastructure, like grids and pipes.

Reply:
okay then, will divide it into 3 separate functions